lib/tesla/middleware/decode_rels.ex

```diff
@@ -46,8 +46,8 @@ defmodule Tesla.Middleware.DecodeRels do
   end
 
   defp rel(item) do
-    Regex.run(~r/\A<(.+)>; rel="(.+)"\z/, item, capture: :all_but_first)
-    |> Enum.reverse()
-    |> List.to_tuple()
+    [url, param] = String.split(item, ";")
```

Comment:
Won't this break with `rel="some;thing"` ?

Reply:
I can add a test to check.